sql/core/benchmarks/DataSourceReadBenchmark-results.txt

```diff
@@ -1,431 +1,438 @@
 ---------------------------------------------------------------------------------------------------------------------------
-ParquetReader Vectorized: DataPageV1                    93             94           1        169.9           5.9       1.0X
-ParquetReader Vectorized: DataPageV2                    93             94           1        169.1           5.9       1.0X
-ParquetReader Vectorized -> Row: DataPageV1             61             62           1        258.0           3.9       1.5X
-ParquetReader Vectorized -> Row: DataPageV2             61             62           1        258.4           3.9       1.5X
+ParquetReader Vectorized: DataPageV1                    85             88           4        185.9           5.4       1.0X
+ParquetReader Vectorized: DataPageV2                    84             86           2        186.5           5.4       1.0X
+ParquetReader Vectorized -> Row: DataPageV1             62             64           1        252.7           4.0       1.4X
+ParquetReader Vectorized -> Row: DataPageV2             62             63           1        253.9           3.9       1.4X
 
-OpenJDK 64-Bit Server VM 17.0.11+9-LTS on Linux 6.5.0-1022-azure
+OpenJDK 64-Bit Server VM 17.0.11+9-LTS on Linux 6.5.0-1023-azure
 AMD EPYC 7763 64-Core Processor
 SQL Single SMALLINT Column Scan:          Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-SQL CSV                                           12200          12203           5          1.3         775.7       1.0X
-SQL Json                                           9813           9854          57          1.6         623.9       1.2X
-SQL Parquet Vectorized: DataPageV1                  101            107           6        156.1           6.4     121.0X
-SQL Parquet Vectorized: DataPageV2                  129            135           6        122.3           8.2      94.9X
-SQL Parquet MR: DataPageV1                         1968           1989          29          8.0         125.1       6.2X
-SQL Parquet MR: DataPageV2                         1913           1916           3          8.2         121.6       6.4X
-SQL ORC Vectorized                                  130            135           6        120.8           8.3      93.7X
-SQL ORC MR                                         1593           1600          10          9.9         101.3       7.7X
-
-OpenJDK 64-Bit Server VM 17.0.11+9-LTS on Linux 6.5.0-1022-azure
+SQL CSV                                           10958          10970          18          1.4         696.7       1.0X
+SQL Json                                          10164          10169           7          1.5         646.2       1.1X
+SQL Json with UnsafeRow                           11113          11137          33          1.4         706.5       1.0X
```

Comment:
So, using UnsafeRow is slower than not using it? Is this a negative effect brought about by saving memory?